src/pages/index.vue

```diff
@@ -1,74 +1,72 @@
-
 <template>
-
-<generic-panel class="bg-red-50">
-  <div class="grid grid-cols-2">
-    <div>
-      <h1>Open Source Software PH</h1>
-      <h2>The leading Open Source Software Community in the Philippines</h2>
-      <button class="btn btn-primary">Join The Movement</button>
-      <button class="btn btn-primary">Help Us Grow</button>
-    </div>
-    <div>
-      <img src="https://loremflickr.com/640/480/nightlife" alt="Night Life">
+  <generic-panel class="bg-red-50 font-primary">
+    <div class="grid grid-cols-2">
+      <div>
+        <h1 class="font-roboto">Open Source Software PH</h1>
```

Comment:
The roboto font shouldn't be used for titles. Kindly modify the `tailwind.css`, in the `body` block set the default font-family to Roboto. While you're at it, kindly remove the existing font family in the `tailwind.css`. Thank you.

Additionally, everything is roboto, except for title headings.